open-metadata-conformance-suite/open-metadata-conformance-suite-client/src/main/java/org/odpi/openmetadata/conformance/client/OpenMetadataConformanceTestReport.java

```diff
@@ -25,12 +26,14 @@
  */
 public class OpenMetadataConformanceTestReport
 {
+    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper().enable(SerializationFeature.INDENT_OUTPUT);
```

Comment:
I would think that we should only add the indents if we think that the serialisation is not going to be used at runtime - as there must be a performance overhead to indent (pretty print) the serialised form. Or is there no appreciable impact?